boule/tests/test_sphere.py

```diff
@@ -146,7 +146,7 @@ def test_normal_gravity_only_rotation():
     )
     # Check normal gravity on the equator
     for height in [1, 2, 3, 4]:
-        expected_value = -1e5 * (omega ** 2) * (radius + height)
+        expected_value = 1e5 * (omega ** 2) * (radius + height)
```

Comment:
Wouldn't this be negative since our convention is that downward accelerations are positive?